cmd/swarm-smoke/upload_and_sync.go

```diff
@@ -47,7 +47,7 @@ func uploadAndSyncCmd(ctx *cli.Context) error {
 		seed = inputSeed
 	}
 
-	randomBytes := testutil.RandomBytes(seed, filesize*1000)
+	randomBytes := testutil.RandomBytes(seed, filesize)
```

Comment:
I think we should revert this. I didn't understand why we wanted to have it smaller in the first place to be honest.